spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -1,6 +1,6 @@
 management:
   security:
-    roles: ADMIN
```

Comment:
@ghillert  Removing this property will leave us to use `ACTUATOR` role which is the default. I believe we want to override this here because of the default `ACTUATOR` role.

IMO, it makes sense to change the `management.security.roles` to `MANAGE` as this will be one less role to set explicitly (as ADMIN) when `management.security.enabled` is set to `true`.